packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -1,35 +1,40 @@
 ---
-title: Connect your SCIM API service to Okta
-excerpt: Use Okta to make SCIM requests to your SCIM API service by creating and configuring an Okta SCIM integration from the Admin Console.
+title: Add a private SCIM integration
+excerpt: Create a private SCIM integration instance. Configure and check the attributes and their corresponding mappings in the Admin Console.
 meta:
   - name: description
-    content: Use Okta to make SCIM requests to your SCIM API service by creating and configuring an Okta SCIM integration from the Admin Console.
+    content: Use Okta to make SCIM requests to your SCIM API service by creating and configuring a private SCIM integration instance from the Admin Console.
 layout: Guides
 ---
 
-This guide teaches you how to configure Okta to make SCIM requests to your SCIM API service.
+Use this guide to create and test a private SCIM integration from your Okta org to your SCIM API service. To test and submit your SCIM integration to the Okta Integration Network (OIN) for public use, see [Submit an integration with the OIN Wizard](/docs/guides/submit-oin-app/scim/main/).
```

Comment:
Should we clarify what is meant by "private" vs "public" integration here at the top?

Reply:
We really don't need this line since the content is mostly in the Overview, so I don't want to repeat the same thing here. It's a requirement for the nutrition facts and automated guide landing page desc.